P3826/P3826.md

```diff
@@ -145,7 +145,7 @@ customizable sender algorithms:
 - `sync_wait`
 - `affine_on`
 
-This would leave users with no standard way to start work on a given execution
+This would leave users with no easy standard way to start work on a given execution
 context, or transition to another execution context, or to execute work in parallel,
 or to wait for work to finish.
 
```

Comment:
this comment for the line 152. I think the sentence starting on the line 152 is not necessarily true. Even if we remove `bulk` we could still add something else instead of it to allow a parallel execution on parallel scheduler. But that likely leads to bigger redesign and will (likely) be met as yet another non-baked design at the last moment. So, I decided to keep it as is.